src/codegen/internal-module-registry-scanner.ts

```diff
@@ -7,6 +7,7 @@ export function createInternalModuleRegistry(basedir: string) {
     .flatMap(dir => readdirRecursive(path.join(basedir, dir)))
     .filter(file => file.endsWith(".js") || (file.endsWith(".ts") && !file.endsWith(".d.ts")))
     .map(file => file.slice(basedir.length + 1))
+    .map(x => x.replaceAll("\\", "/"))
```

Comment:
Can we use the `path.normalize(x).split(path.delimiter).join("/")` method here? Maybe a convoluted case. Thinking more in a `node:path`- way.

Reply:
this feels, overly convoluted

and it will be harder to read.

Reply:
Ya, looking for more of a https://github.com/wessberg/crosspath kinda thing to simplify a lot of our normalize to posix stuff.